source/providerdoall.c

```diff
@@ -109,11 +108,10 @@ int main(int argc, char *argv[])
         goto err;
     }
 
-    ttime = times[0];
-    for (i = 1; i < threadcount; i++)
-        ttime = ossl_time_add(ttime, times[i]);
+    for (i = 0; i < threadcount; i++)
```

Comment:
nit: comparison of integer expressions of different signedness

Reply:
Both `i` and `threadcount` are an `int`